src/components/Player/Pages/Wardmap/Wardmap.jsx

```diff
@@ -9,21 +9,25 @@ import Heatmap from '../../../Heatmap';
 import Container from '../../../Container';
 
 const MAX_WIDTH = 1200;
+const HALF_WIDTH = 580;
```

Comment:
I wonder if we can make these percentages so we can adjust the width of the container without modifying this?